files/templates/comments.html

```diff
@@ -160,7 +160,7 @@ <h5 class="pop-username text-truncate text-black"></h5>
 	{% if not isreply %}
 		<div class="comment-collapse-icon" onclick="collapse_comment('{{c.id}}', this.parentElement)"></div>
 		<div class="comment-collapse-bar-click" onclick="collapse_comment('{{c.id}}', this.parentElement)">
-			<div class="comment-collapse-bar"></div>
```

Comment:
Alright I may be confused about what's going on here :V

It looks like this adds an attribute to the bar with the current level. Then there's a JS function, `colorize`, which I think doesn't use this attribute but calculates the level separately. I'm not sure anything uses this attribute.

Couldn't the JS function be removed entirely, and this code changed to just apply the `collapse-bar-{{level}}` class?

Lemme know what's going on here :)

Reply:
(The more I work on this issue, the more I'm reconsidering it as a good idea. If you think this is a feature we actually need, this PR is a good start, but you're not going to hurt my feelings if you scrap it)

The way colorize works:

* When colorize is run with no arguments, it looks for every collapse bar, gets the depth attribute, and sets the color. If for some reason the collapse bar doesn't have the attribute, it manually calculates the depth and sets it that way.
* When colorize is run with an element argument, it finds a collapse-bar child of that element and calculates a depth for just that one.

The reason you can run it in two different "modes" is so that it doesn't have to recolor the entire page when you click a `More Comments` button in a thread- that button loads in a bunch of nested replies and then runs colorize on each of them individually.

The other time that colorize is run is when you try to load in another page of top level comments. I couldn't find any easy way to restrict colorization to just the new comments, so I ran it over the entire page again. It updates the data classes (which are wrong for dynamically loaded comments) as it goes, so it should only have to do a manual depth calculation for the new comments- the old ones, it just sets the color again. 

So one `document.getElementsByClassName` query and then manual depths for each of the newly inserted elements, basically.

Reply:
Alright, that's reasonable.

Lemme look at this locally and see if I can find a set of colors that I think is a good idea; this may take a day or two :) If it turns out I don't think it's visually an improvement then yeah we might can it, it is getting kinda complicated.

Reply:
Yeah, no worries. Let me know if you need anything else.